packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -70,14 +70,23 @@ class CupertinoIcons {
   static const IconData flag = const IconData(0xf42c, fontFamily: iconFont, fontPackage: iconFontPackage);
 
   /// A magnifier loop outline.
-  static const IconData search = const IconData(0xf4c6, fontFamily: iconFont, fontPackage: iconFontPackage);
+  static const IconData search = const IconData(0xf4a5, fontFamily: iconFont, fontPackage: iconFontPackage);
 
   /// A checkmark.
-  static const IconData check_mark = const IconData(0xf41e, fontFamily: iconFont, fontPackage: iconFontPackage);
+  static const IconData check_mark = const IconData(0xf3fd, fontFamily: iconFont, fontPackage: iconFontPackage);
 
   /// A checkmark in a circle.
-  static const IconData check_mark_circled = const IconData(0xf41f, fontFamily: iconFont, fontPackage: iconFontPackage);
+  static const IconData check_mark_circled = const IconData(0xf3fe, fontFamily: iconFont, fontPackage: iconFontPackage);
 
   /// A thicker left chevron used in iOS for the nav bar back button.
-  static const IconData back = const IconData(0xf3f0, fontFamily: iconFont, fontPackage: iconFontPackage);
+  static const IconData back = const IconData(0xf3cf, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Outline of a simple front-facing house.
+  static const IconData home = const IconData(0xf447, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A right facing shopping cart outline.
+  static const IconData shopping_cart = const IconData(0xf3f7, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// 3 solid dots.
+  static const IconData ellipsis = const IconData(0xf46a, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
is there no way we can automatically generate the complete set here?

Also, it would be really nice if we could have the docs show the icons, like we do for Material icons.

Reply:
I'll try to figure out some way to show the image in the doc. 

I think it'll be hard to automate the semantic description of the font though. But I'll probably put the mapping in the font package like the material font the next time I update the fonts and add some doc.